### PR TITLE
Bumps the ES-API to 6.2.4

### PR DESCRIPTION
**Pull request by Tobsucht**
Adjust to Java-API-changes, change scripting language to painless, provide a alternative to the _all-field
by using a custom-_all field using the copy_to attribute.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -37,12 +37,8 @@
         <dependency>
             <groupId>org.elasticsearch.client</groupId>
             <artifactId>transport</artifactId>
-            <version>5.4.0</version>
+            <version>6.1.0</version>
             <exclusions>
-                <exclusion>
```
Comment (Tobsucht) [inline pom.xml]: If we use the exclusion, ES cannot find the class NettyRuntime

File: src/main/java/sirius/search/ForeignKey.java [hunk 2]
```diff
@@ -378,7 +378,7 @@ private Script computeUpdateScript(Entity parent, boolean updateParent) {
         for (Reference ref : references) {
             sb.append("ctx._source.");
             sb.append(ref.getLocalProperty().getName());
-            sb.append("=");
+            sb.append("=params.");
```
Comment (Tobsucht) [inline src/main/java/sirius/search/ForeignKey.java:381]: painless needs the explicit scope of the variables

File: src/main/java/sirius/search/ForeignKey.java [hunk 3]
```diff
@@ -40,7 +40,7 @@
  */
 public class ForeignKey {
 
-    private static final String LANGUAGE_GROOVY = "groovy";
+    private static final String LANGUAGE_PAINLESS = "painless";
```
Comment (Tobsucht) [inline src/main/java/sirius/search/ForeignKey.java:43]: everything beside painless is deprecated

**Review by meg1502 — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (Tobsucht) [reply pom.xml]: wrong => use same netty version within sirius-web and transportclient

**Review by jmuscireum — APPROVED** [review]

**Pull request merged**